scripts/slurm_submit

```diff
@@ -3,9 +3,13 @@
 # Wrapper script for Slurm batch submit
 # Args are submit script, outputfile, and error file
 # Return the job id via stdout
+# Ensure that  slurm_submit and slurm_checkjob are in the PATH . ~/.local_settings may need to be updated 
+
+module purge
+PATH=/usr/common/software/bin:/usr/common/mss/bin:/usr/common/nsg/bin:/opt/ovis/bin:/opt/ovis/sbin:/usr/local/bin:/usr/bin:/bin:/usr/lib/mit/bin:/usr/lib/mit/sbin
```

Comment:
I generally don't think paths should be hardcoded in scripts, but it seems unlikely this script is ever going to be used for anything other than hipmer so it's probably fine